.ci/test_r_package.sh

```diff
@@ -147,7 +147,7 @@ elif [[ $R_BUILD_TYPE == "cran" ]]; then
     # on Linux, we recreate configure in CI to test if
     # a change in a PR has changed configure.ac
     if [[ $OS_NAME == "linux" ]]; then
-        ${BUILD_DIRECTORY}/R-package/recreate-configure.sh
+        "${BUILD_DIRECTORY}"/R-package/recreate-configure.sh
```

Comment:
```suggestion
        "${BUILD_DIRECTORY}/R-package/recreate-configure.sh"
```

Quote the full path on every one of these, please.